### DISCUSSION TITLE
How can I see the change difference?

### DISCUSSION THREAD

**Discussion by antimora** [zed-industries/zed#7701] [Feature Requests]
I can see what line changed but how can I see what was the previous one for reverting purposes?

Here is example: 
<img width="758" alt="image" src="https://github.com/zed-industries/zed/assets/939125/113c32aa-3006-4e7b-8587-05fd70104dc9">

**Comment by SomeoneToIgnore** [comment]
Zed has nothing on that front yet, unfortunately.
